.gitignore

```diff
@@ -21,6 +21,7 @@
 .libs
 .project
 .settings
+.idea
```

Comment:
nitpick - I've found its fine to do this with my local users gitignore instead of inject here ... otherwise we could add a quite a few more

Reply:
will remove. found some .vs stuff there and thought...